### PR TITLE
CI: add build configuration for P2 E2E tests.

### LINKED ISSUE #56664: Playwright: add e2e tests for P2
### Details

Investigate feasibility of adding automated e2e tests for P2s as part of the Gutenberg update process.

Context: p1632936169323300-slack-C7YPUHBB2

### Checklist

- [ ] Set up TeamCity task for P2 tests.
- [ ] Set up a testing P2 blog and user account.
- [ ] Set up a cron job to clear the test P2.
- [ ] Write test cases for the front-end editor.

### Related

_No response_

### PR DESCRIPTION
**Pull request by worldomonation**
#### Changes proposed in this Pull Request

This PR builds on top of #58082 to actually schedule the P2 E2E runs on a regular basis.

Key changes:
- addition of P2E2E build configuration under Calypso > Web app.
- schedule task to run once every 3 hours, every day of the week.

Related to #58082, #56664.

### CODE REVIEW

**Comment by scinos** [comment]
> addition of P2E2E build configuration under Calypso > Web app.

What's the rationale for using that group? In my mind, this belongs in the `WPCom Tests` group, as the editor lives mostly outside calypso

**Comment by worldomonation** [comment]
> > addition of P2E2E build configuration under Calypso > Web app.
> 
> What's the rationale for using that group? In my mind, this belongs in the `WPCom Tests` group, as the editor lives mostly outside calypso

No particular reason to this. My rationale was that being under Webapp would make it more visible to me as I am constantly monitoring the build configurations there.

I'll move it into WPCOM Tests for now.

**Comment by worldomonation** [comment]
Moved into WPCOM Tests and ready for review.

File: .teamcity/_self/projects/WPComTests.kt [hunk 1]
```diff
@@ -836,3 +839,137 @@ private object I18NTests : BuildType({
...
+				githubUrl = "https://api.github.com"
+				authType = personalToken {
+					token = "credentialsJSON:57e22787-e451-48ed-9fea-b9bf30775b36"
+				}
+			}
+		}
+	}
+
+	triggers {
+		vcs {
```
Comment (scinos) [inline .teamcity/_self/projects/WPComTests.kt]: I'm running with the assumption that those tests are testing WPCOM features and not Calypso features, let me know if that's not correct.

But if my assumption is true, then we don't need to automatically run them on branch changes.

File: .teamcity/_self/projects/WPComTests.kt [hunk 2]
```diff
@@ -836,3 +839,137 @@ private object I18NTests : BuildType({
...
+				buildRule = lastSuccessful()
+			}
+		}
+	}
+
+	dependencies {
+		snapshot(BuildDockerImage) {
+			onDependencyFailure = FailureAction.FAIL_TO_START
+		}
+	}
```
Comment (scinos) [inline .teamcity/_self/projects/WPComTests.kt]: As above, if the tests are not related to Calypso, we don't need to wait for the docker image to be built.

File: .teamcity/_self/projects/WPComTests.kt (hunk 1 above)
Comment (worldomonation) [reply .teamcity/_self/projects/WPComTests.kt]: There's been more build config differences than I expected between WPCOM and Calypso. I had been glossing over it, copy-pasting the config but that'll no longer cut it.

wrt to triggers: yes, I think it's okay leaving it to run on a schedule for the time being. If we see more P2 errors on the frontend that aren't getting caught, then perhaps we'd change the trigger to run on every trunk.

Comment (worldomonation) [reply .teamcity/_self/projects/WPComTests.kt]: vcs trigger removed in d9074c6d975711803c45dc482531f935bce9cae3.

File: .teamcity/_self/projects/WPComTests.kt (hunk 2 above)
Comment (worldomonation) [reply .teamcity/_self/projects/WPComTests.kt]: Imports and docker dependencies removed in d9074c6d975711803c45dc482531f935bce9cae3.

File: .teamcity/_self/projects/WPComTests.kt [hunk 3]
```diff
@@ -836,3 +837,124 @@ private object I18NTests : BuildType({
...
+	""".trimIndent()
+
+	vcs {
+		root(Settings.WpCalypso)
+		cleanCheckout = true
+	}
+
+	params {
+		checkbox(
+			name = "env.SAVE_AUTH_COOKIES",
```
Comment (worldomonation) [inline .teamcity/_self/projects/WPComTests.kt:857]: I've left this param in here for now but toggled the value to 'false'.

**Comment by worldomonation** [comment]
Sorry for the back and forth. I've checked over the configuration twice and it should no longer include unnecessary things.

**Review by scinos — APPROVED** [review]

**Pull request merged**